### PR TITLE
Created Contributing.md

### PR DESCRIPTION
**Pull request by codydaig**
This is replacing #690. While squashing, I had some git issues. 

@ilanbiala All your requested changes have been made.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# How to contribute
+
+Support and contributons from the open source community are essential for keeping
```
Comment (rhutchison) [inline CONTRIBUTING.md]: spelling: contributions

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+MEAN.js up to date and always improving! There are a few guildlines that we need
+contributors to follow to keep the project consistent, as well as allow us to keep
+maintaining MEAN.js in a reasonable amount of time.
+
+## Getting Started
+
+Before you begin making changes:
+* Submit an Issue on Github, assuming one does not already exist.
```
Comment (rhutchison) [inline CONTRIBUTING.md]: Link to issues?

**Comment by codydaig** [comment]
@rhutchison Made your requested changes.

**Comment by rhutchison** [comment]
LGTM @lirantal @ilanbiala 

Thank you @codydaig

**Comment by lirantal** [comment]
yeah let's go with this for now until I have some more free time to add my ideas to the CONTRIBUTING area.

**Comment by lirantal** [comment]
Thanks @codydaig

**Pull request merged**